src/status_im2/contexts/wallet/events.cljs

```diff
@@ -72,35 +75,54 @@
 
 (rf/reg-event-fx
  :wallet/save-account
- (fn [_ [{:keys [account callback]}]]
+ (fn [_ [{:keys [account on-success]}]]
```

Comment:
`callback` is too generic, actually, that `callback` is only executed when the rpc call was successful, so a better name is `on-success`